docs/synapse/userguides/ug015_storm_ref_pivot.rst

```diff
@@ -393,7 +397,7 @@ Optional parameters:
   * To disable limits on recursion (e.g., continue executing pivots until no more results are returned), ``recurnlim`` should be set to 0 (``recurnlim=0``).
   * **Note:** due to a known bug, ``tree()`` currently ignores any ``recurnlim`` parameter.
 
-* **Return limit:** specify the maximum number of nodes returned by the ``tree()`` query.
+* **Return limit:** specify the maximum number of nodes added to the current working set by the ``tree()`` query.
```

Comment:
Suggest also updating "Usage Notes" to parallel join(), refs() - e.g.:
* tree() does not consume nodes, so the results of a tree() operation will include both the original working set as well as the resulting set of nodes.
... stuff ...
Because tree() does not consume nodes, this impacts the results returned by either the limit= parameter of the limit() operator.
... stuff ...

Reply:
fixed